modules/doc/content/framework_development/authorship_guidelines.md

```diff
@@ -0,0 +1,74 @@
+# MOOSE papers authorship guidelines
+
+MOOSE is a collaborative effort and not only do we always welcome contributions, but also want to
+credit them appropriately and in a sustainable manner. We do this by including our contributors
+in the relevant journal article(s) that should be used by all, both within the community and externally,
+to cite the MOOSE framework and its modules. Please see the [citing.md] page for the current versions
```

Comment:
the "and its modules" part might be problematic. We want modules to be cited individually. 

Prefer
- cite the MOOSE framework.
- cite MOOSE
?

Reply:
What about "cite the MOOSE framework and any modules used" ?

Reply:
I get what you mean that using a module you should still cite moose framework 

But citing a moose module externally from a publication that did not use it, you probably don't need to cite moose framework (though it does not hurt)

Reply:
oh nevermind this is a paragraph on both moose and its modules. this works